proxy-robots-module/src/main/java/orwell/proxy/config/ConfigScout.java

```diff
@@ -0,0 +1,56 @@
+package orwell.proxy.config;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlType;
+
+@XmlType(propOrder = {"camera", "image"})
+public class ConfigScout implements IConfigRobot {
```

Comment:
This name does not seem to be very explicit about what this class does. It is reading the configuration from an XML file is it not ?